shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/java/org/apache/shardingsphere/distsql/parser/core/DistSQLVisitor.java

```diff
@@ -372,7 +373,12 @@ public ASTNode visitShowResources(final ShowResourcesContext ctx) {
     public ASTNode visitShowRule(final ShowRuleContext ctx) {
         return new ShowRuleStatement(ctx.ruleType().getText(), null == ctx.schemaName() ? null : (SchemaSegment) visit(ctx.schemaName()));
     }
-    
+
+    @Override
+    public ASTNode visitShowShardingBindingTableRules(final DistSQLStatementParser.ShowShardingBindingTableRulesContext ctx) {
```

Comment:
Please remove `DistSQLStatementParser.` and just import `ShowShardingBindingTableRulesContext`. I still notice such an issue shows up many times. Please consider public it as a community task.

Reply:
I have created issue.